app/src/main/res/xml/preferences.xml

```diff
@@ -76,6 +76,12 @@
     <PreferenceCategory android:title="@string/playback_pref">
         <de.danoeh.antennapod.preferences.SwitchCompatPreference
             android:defaultValue="true"
+            android:enabled="false"
```

Comment:
Is there any value in enabling sonic by default?  Or will that mess with people?